src/main/java/racingcar/Race.java

```diff
@@ -0,0 +1,49 @@
+package racingcar;
+
+public class Race {
+
+    public static void execute() {
+        Car[] cars = IOClass.getName();
+        if (!IOClass.checkCarName(cars)) {
+            System.out.println("자동차 이름은 5자를 초과할 수 없습니다.");
+            return;
+        }
+
+        Race race = new Race();
+        int times = IOClass.getTimes();
+        repeatGame(cars, times);
+        IOClass.printWinners(cars);
+        return;
+    }
+
+    public static int getMaxScore(Car[] cars) {
+        int maximum = 0;
+        for (int i = 0; i < cars.length; i++) {
+            maximum = Math.max(maximum, cars[i].getPosition());
+        }
+        return maximum;
+    }
+
+    public static void repeatGame(Car[] cars, int times) {
+        while ((times--) > 0) {
```

Comment:
```suggestion
        for(int j = 0; j < times; j++) {
```
조금 더 직관적인 for문으로 변경해보면 어떨까요? 🤔